Topics/Tech_Stacks/SCSS.md

```diff
@@ -0,0 +1,329 @@
+# Learning SCSS, a CSS Preprocessor
+
+## Table of contents
+
+### [Introduction](#introduction-1)
+
+### [Getting Started](#getting-started-1)
+
+### [Variables](#variables-1)
+
+### [Maps](#maps-1)
+
+### [Nesting](#nesting-1)
+
+### [Partials](#partials-1)
+
+### [Functions](#functions-1)
+
+### [Placeholder Selectors and Extend/Inheritance](#placeholder-selectors-and-extendinheritance-1)
+
+### [Operators](#operators-1)
+
+### [Compiling files to CSS](#compiling-scss-files-into-css-files)
+
+### [References](#references-and-links)
+
+## Introduction
+
+SCSS is a programming language that extends the capabilities of CSS by introducing new syntax to streamline the process of creating stylesheets. SCSS is a superset of CSS which means that all valid CSS code is also valid SCSS code. Due to this, it is recommended that before you start learning SCSS you should have some working knowledge of CSS. You can get started at [this CSS guide](./CSS.md), but it is suggested to have more practice with CSS. Although not required, it is also recomended to use the BEM methodology when creating class names for HTML elements. You can find more information about BEM on [their website here](https://getbem.com/).
```

Comment:
This is a solid introduction and I know below you showed the difference between CSS and SCSS but I think it be beneficial to give the user a quick snapshot of the differences between CSS and SCSS and explain why SCSS may be better.

Reply:
I have made the changes. Let me know what you think and If I should include more or not.

Reply:
I like this better, gives me a sense of why I should care about SCSS/what I could gain from it 👍